core/base-service/errors.js

```diff
@@ -72,6 +72,23 @@ class Inaccessible extends ShieldsRuntimeError {
   }
 }
 
+class ImproperlyConfigured extends ShieldsRuntimeError {
```

Comment:
Do you think we'd ever have other, non-auth misconfiguration types of "configuration" related errors? IMO this is good as is, just wanted to put the question out there.

Reply:
Hmm, I'm not sure. Usually we quit on bad configuration. What makes this one unusual is that we have started the server, and just aren't supporting this request.

I borrowed the name [from Django](https://docs.djangoproject.com/en/2.2/ref/exceptions/#improperlyconfigured) (which admittedly has a lot of configuration).

Reply:
No worries, was wondering if we'd ever need/want different error types for Auth Configuration errors vs. other config errors, but doesn't seem necessary at all considering.